articles/screenshot-stabilization/index.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: "Stabilize Flaky Tests for Visual Testing"
+description: "Explore common use-cases responsible for flaky tests in visual testing and best practices to resolve them."
+category: Visual testing
+author: Jeremy Sfez
+date: 2024-02-15
+image: ./main.jpg
+imageAlt: Image not loaded
+---
+
+Visual testing is the most efficient way to test your app as it's **fast** and **robust** (doesn't require maintenance). Yet, flaky tests create frustration and erode developers' trust in their testing suite, potentially leading to the abandonment of visual testing altogether. In this article, I'll describe the **common culprits responsible for flaky screenshots and how to fix them**. I'll also introduce [Argos](https://argos-ci.com)'s solutions to stabilize screenshots.
+
+<MainImage />
+
+## What is a Flaky Test?
+
+In visual testing, a flaky test produces screenshots that differ randomly between test runs without any changes to the code. The common causes are often **network latency, lazy loading, animations, and dynamic content like dates and external scripts**.
+
+#### Disclaimer: A Flaky Test Means a Flaky UI
```

Comment:
I would be way more impactful to cite Guillermo Rauch here. 
https://rauchg.com/2020/2019-in-review#flaky-tests-flaky-ux